sysconfig/bb5/users/packages.yaml

```diff
@@ -137,10 +137,6 @@ packages:
         paths:
             pkg-config@0.27.1: /usr
         version: [0.27.1]
-    python:
-        paths:
-            python@2.7.13: /usr
-        version: [2.7.13]
```

Comment:
This python is used by other packages like NEURON. I am aware of issue while installing py-cythong. Until we get our own python installation, could you guys use python version other than 2.7.13? i.e. explicitly specify something like `spack install steps ^python@2.7.14` ?